docs/Addition-Guide.md

```diff
@@ -0,0 +1,23 @@
+# What if I want software added?
+
+Are you missing your favorite software in Homebrew? Then you're the perfect person to resolve this problem.
+
+But before you do that, please check if nobody else beat you to the punch by checking the [homebrew-core](https://github.com/Homebrew/homebrew-core/pulls) or [linuxbrew-core](https://github.com/Homebrew/linuxbrew-core/pulls) pull request list.
+
+Nobody did? Good, you're still the best person to resolve this. 
+
+## Before you begin
+Check if the software is acceptable for homebrew by reading the [requirements](Acceptable-Formulae.md). If the software is a version of already accepted software, read the [version requirements](Versions.md).
+
+## Writing the formula
+1. It's a good idea to look up some brewed software that is similar to your favorite software in language and/or build method and look at the formula code for that to gather some inspiration.
```

Comment:
Markdown trick: If you number these all as `1.` then they'll auto-number when rendered. So if you add a step in the middle you don't have to manually increment each one.